conf/mesos-site.xml

```diff
@@ -19,54 +19,84 @@
 <configuration>
   <property>
     <name>mesos.hdfs.data.dir</name>
+    <description>The primary data directory in HDFS</description>
     <value>/tmp/hdfs/data</value>
   </property>
   
   <property>
     <name>mesos.hdfs.secondary.data.dir</name>
+    <description>The secondary data directory in HDFS</description>
     <value>/var/run/hadoop-hdfs</value>
   </property>
-  
+
+  <property>
+    <name>mesos.hdfs.native-hadoop-binaries</name>
+    <description>Mark true if you have hadoop pre-installed on your host machines (otherwise it will be distributed by the scheduler)</description>
+    <value>false</value>
+  </property>
+
   <property>
     <name>mesos.hdfs.framework.mnt.path</name>
+    <description>Mount location (if above config is marked false)</description>
     <value>/opt/mesosphere</value>
   </property>
 
   <property>
     <name>mesos.hdfs.state.zk</name>
     <value>localhost:2181</value>
+    <description>Hostname and port of zookeeper node location</description>
```

Comment:
Sure, but another point of multiple zk nodes is that you have backup zks to contact if the first one is down. It's not just about a voting quorum.